multicast/hear.py

```
@@ -244,6 +252,236 @@
+
+		Testcase 1: McastServer should be automatically imported.
+
+			>>> multicast.hear.McastServer.__name__ is not None
+			True
+			>>> multicast.hear.McastServer.__log_handle__ is not None
+			True
+			>>>
+
+	"""
+
+	def __init__(self, server_address, RequestHandlerClass, bind_and_activate=True) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `server_address`

(ANN001)